files/build_templates/per_namespace/syncd.service.j2

```diff
@@ -0,0 +1,41 @@
+[Unit]
+Description=syncd service
+{% if multi_instance == 'true' %}
+Requires=database@%i.service
+Requires=updategraph.service
+After=database@%i.service swss@%i.service
+   {% if sonic_asic_platform == 'vs' %}
+Requires=topology.service
+After=topology.service
+   {% endif %}
+{% else %}
+Requires=database.service
+After=database.service swss.service
+{% endif %}
```

Comment:
This could further be simplified to:

```suggestion
Requires=database{% if multi_instance == 'true' %}@%i{% endif %}.service
After=database{% if multi_instance == 'true' %}@%i{% endif %}.service
After=swss{% if multi_instance == 'true' %}@%i{% endif %}.service
{% if multi_instance == 'true' and sonic_asic_platform == 'vs' %}
Requires=topology.service
After=topology.service
{% endif %}
```

Also not that I removed `Requires=updategraph.service` because it also appears on line 20.

Reply:
Fixed as per comment.